prow/test/integration/config/prow/config.yaml

```diff
@@ -22,7 +22,9 @@ gerrit:
   org_repos_config:
   - org: http://fakegerritserver
     repos:
-    - gerrit-test-infra
+    - gerrit-test-infra-0
```

Comment:
nit: in the future might want to consider naming these by test case names instead of numbers, so that it will be easier to figure out which one is used where